x1s.json

```diff
@@ -0,0 +1,24 @@
+{
+  "response": [
+    {
+        "maintainer": "xBrownieCodez",
+        "oem": "Samsung",
+        "device": "Samsung Galaxy S20 / Samsung Galaxy S20 (5G)",
```

Comment:
Remove Samsung from device name
Reason: device name is kind of big as you seem to want to support both normal and 5G version
This will make website truncate name resulting in weird description